### PR TITLE
[SPCS][User Metrics] Updating the Metrics docs

### PR DESCRIPTION
**Pull request by sfc-gh-tkommineni**

### CODE REVIEW

File: user-metrics/metrics-service-datadog/metrics-service.yml.template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 spec:
   container:
     - name: mdservice
-      image: ${SPCS_IMAGE_REPO}/mdservice:v0.0.1
+      image: SPCS_IMAGE_REPO/mdservice:VERSION
```
Comment (sfc-gh-aivanou) [inline user-metrics/metrics-service-datadog/metrics-service.yml.template:4]: is it a right change?

File: user-metrics/metrics-service-datadog/metrics-service.yml.template [hunk 2]
```diff
@@ -25,10 +25,3 @@ spec:
         limits:
           memory: 200M
           cpu: 200m
-  endpoints:
```
Comment (sfc-gh-aivanou) [inline user-metrics/metrics-service-datadog/metrics-service.yml.template:28]: why do we delete these endpoints?

File: user-metrics/metrics-service-datadog/metrics-service.yml.template (hunk 1 above)
Comment (sfc-gh-tkommineni) [reply user-metrics/metrics-service-datadog/metrics-service.yml.template:4]: yes verified.
The reason this has to be done is customers don't have gettext by default on their machine
using sed to replace it

File: user-metrics/metrics-service-datadog/metrics-service.yml.template (hunk 2 above)
Comment (sfc-gh-tkommineni) [reply user-metrics/metrics-service-datadog/metrics-service.yml.template:28]: This is a datadog example and these endpoint will not have any relevance hence removed them

**Review by sfc-gh-aivanou — APPROVED** [review]

**Pull request merged**